static/app/views/replays/detail/layout/fluidHeight.tsx

```diff
@@ -1,11 +1,11 @@
 import styled from '@emotion/styled';
 
-const FluidHeight = styled('div')`
+const FluidHeight = styled('div')<{overflow?: 'hidden' | 'auto' | 'scroll'}>`
   display: flex;
   flex-direction: column;
   flex-wrap: nowrap;
   flex-grow: 1;
-  overflow: hidden;
+  overflow: ${p => p.overflow ?? 'hidden'};
```

Comment:
I think we should create a new styled component as needed instead of adding a dynamic rule as the latter has a bit of perf impact.